stores/moderationSearchHealthcareProfessionalsStore.ts

```diff
@@ -0,0 +1,49 @@
+import { gql } from 'graphql-request'
```

Comment:
Hmm just a thought, but do we need this store? We already have a `healthcareProfessionalsStore` that these could be added to.

Reply:
It's tied to this:

![image](https://github.com/user-attachments/assets/75af1339-40b6-4f08-97ec-af996c73dd80)

At the moment the functionality how it's designed in the UI is not yet possible. But I can see that this would change in the future. So with that I thought for long term.

@theyokohamalife This only would be possible if we add an Id filter, what I actually wanted to ask about. Would it not be cleaner if we just have one searchHealthcareProfessional and add the id as filter in the backend?

Reply:
I guess what I'd propose is defining a `getHealthcareProfessionalById` function in the `healthcareProfessionalsStore.ts`. I think that we should write our queries and mutations in a way that they aren't tied to specific components and can be used throughout the app.

Reply:
> I guess what I'd propose is defining a getHealthcareProfessionalById function in the healthcareProfessionalsStore.ts. I think that we should write our queries and mutations in a way that they aren't tied to specific components and can be used throughout the app.

this makes sense to me. Your point @Anissa3005 that we MAY have additional functions for moderation in the future, is a good design example where we shouldn't build for something that may not happen. The reason is if it doesn't, we've overdesigned/added complexity. It's a habit that when multiplied x50 devs, the codebase becomes more complex than it needs. You can always split it out later into it's own section later, but it's hard to do the opposite direction and consolidate.